components/Image/modules/resizeWithFastly.js

```diff
@@ -0,0 +1,32 @@
+import { LAYOUT } from '../../../common';
+
+const AKAMAI_DOMAIN = 'imgs-akamai.mnstatic.com';
+const LOW = { W: 375, H: 736 };
+const REGULAR = { W: 1024, H: 768 };
+const HIGH = { W: 1920, H: 1200 };
+
+export default ({ uri = '', ratio = 1, width = null, height = null }) => {
```

Comment:
how about some tests?

Reply:
A test for proper url creation when width and height are requested included